### PR TITLE
[REF] base_import_module_group: Remove not working auto-install

### PR DESCRIPTION
**Pull request by SirTakobi**

### CODE REVIEW

**Review by thomaspaulb — APPROVED** [review]

File: base_import_module_group/__manifest__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "auto-install": True,
+    "auto_install": True,
```
Comment (thomaspaulb) [inline base_import_module_group/__manifest__.py]: Actually, no. Why should this module be `auto_install` anyway? Let's just remove the whole key instead.

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: base_import_module_group/__manifest__.py (hunk 1 above)
Comment (SirTakobi) [reply base_import_module_group/__manifest__.py]: Seems reasonable: no-one will be missing a feature that doesn't work

**Review by thomaspaulb — APPROVED** [review]

**Comment by SirTakobi** [comment]
@OCA/tools-maintainers can this be merged? Thanks!

**Review by dreispt — APPROVED** [review]

**Pull request merged**